### PR TITLE
Add changelog for 5.8.0 release

### PR DESCRIPTION
**Pull request by will-moore**
Covers PRs listed at https://github.com/ome/omero-web/milestone/2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+   - Fix 'Move to Group' when data owner not in origin group ([#212](https://github.com/ome/omero-web/pull/212))
+   - History results page supports browse to data ([#206](https://github.com/ome/omero-web/pull/206))
+   - Fix shortening of Companion file names ([#198](https://github.com/ome/omero-web/pull/198))
+   - Fix pagination in history page and omero_table ([#203](https://github.com/ome/omero-web/pull/203))
+   - Link ownership matches child owner ([#199](https://github.com/ome/omero-web/pull/199))
+   - Improve download of OMERO.table as csv performance ([#192](https://github.com/ome/omero-web/pull/192))
+   - Fix display of disk usage for very small percentages ([#211](https://github.com/ome/omero-web/pull/211))
+
+- API changes:
+   - @login_required(doConnectionCleanup=False) will close conn unless streaming ([#191](https://github.com/ome/omero-web/pull/191))
```
Comment (jburel) [inline CHANGELOG.md:15]: this could probably be rephrased "will close conn``

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+   - Return 404 for imgData if image not found ([#209](https://github.com/ome/omero-web/pull/209))
```
Comment (jburel) [inline CHANGELOG.md:16]: Same

**Review by manics — APPROVED** [review]
Looks fine to me

**Comment by will-moore** [comment]
@jburel Does that commit address your points?
If so, can you merge? Thanks.

**Review by jburel — APPROVED** [review]

**Pull request merged**